src/app/pages/overlays/complex-bottom-sheet/bottom-sheet/bottom-sheet.scss

```diff
@@ -1,22 +1,19 @@
+.mat-h3 {
+    margin-bottom: 0;
+}
+
+.active * {
+    color: #007bc1;
```

Comment:
Can we pull this from the colors package

Reply:
Absolutely :)